pkg/operator/operatorclient.go

```diff
@@ -0,0 +1,36 @@
+package operator
+
+import (
+	"fmt"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/cache"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
```

Comment:
nit: whitespace in imports (really a very small nit)

Reply:
Let's continue this discussion here: https://github.com/openshift/cluster-policy-controller/pull/61#discussion_r654274660